cypress/support/commands.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="cypress" />
+
+import '@testing-library/cypress/add-commands';
+
+Cypress.Commands.add('checkToastMessage', (type: 'success' | 'error', message: string) => {
```

Comment:
How about naming it:

```suggestion
Cypress.Commands.add('hasToastMessage', (type: 'success' | 'error', message: string) => {
```

The following reads well:

```
cy.hasToastMessage('error', 'An issue occurred when creating the user account');
```